### ISSUE TITLE
Optimizing transition animations

### ISSUE THREAD

**Issue by KEVIN970712** [mostafaalagamy/Metrolist#183]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

I am a user who switched from innertune. This app is good but the animation is much stiffer than the former. I hope it can continue to be optimized.
By the way, and I wish there was a switch to disable predictive back.

### Why do you want this feature?

For a better user experience

### Additional information

https://github.com/user-attachments/assets/23813ada-46d9-43ea-9d7a-d3c98eb8fc4b

**Comment by ongiirc** [comment]
Yeah I can agree But metroList has way Better UI in terms of the "expanded player" or the "main player screen" whatever you call it, Especially when you use blur, it looks stunning compared to innertune

**Comment by mahoraga40** [comment]
I am also a user who switched from Innertune. Ngl Metrolist has clean black&white cool UI. But the only drawback is, the **animations** in some specific segments feels laggy (btw I'm a 120hz user). I don't know the fix but I hope it can continue to be optimized. Also thanks for adding the yt sync option!!

**Comment by ongiirc** [comment]
@mahoraga40 Yeah I agree It kinda feels laggy On My Phone too. But let me correct You in the 120hz part, If you switch to A higher refresh rate Your phone's only gonna Use it in System apps or When you're in the Launcher (the system launcher) But when you open third party apps (like Metrolist) it automatically Switches to 60hz. so you're Still using 60hz in Metrolist. But yes I can Tell that Metrolist feels laggy on my Phone (kinda) But the player UI is the best I've ever seen (I used innertune too)

**Comment by mostafaalagamy** [comment]
Has the application's performance improved after the last commit was pushed?

**Comment by ongiirc** [comment]
@mostafaalagamy Yeah now it doesn't Seem to lag no More. I'd still like that "Fade" kinda animation From innertune Because absolutely NO menu haze ANIMATION is kinda boring. I'd say performance is good now.

**Comment by mahoraga40** [comment]
> [@mahoraga40](https://github.com/mahoraga40) Yeah I agree It kinda feels laggy On My Phone too. But let me correct You in the 120hz part, If you switch to A higher refresh rate Your phone's only gonna Use it in System apps or When you're in the Launcher (the system launcher) But when you open third party apps (like Metrolist) it automatically Switches to 60hz. so you're Still using 60hz in Metrolist. But yes I can Tell that Metrolist feels laggy on my Phone (kinda) But the player UI is the best I've ever seen (I used innertune too)

You're right, when i tested this by enabling the option called "show refresh rate" it show that Metrolist app goes from 120hz to 60hz. What I mean is it fluctuates between 120-60hz. All though i found no more problems. Great App!
